lib/maps_launcher.dart

```diff
@@ -56,20 +56,33 @@ class MapsLauncher {
     return uri;
   }
 
+  static Future<LaunchMode> _launchMode() async =>
+      await supportsLaunchMode(LaunchMode.externalNonBrowserApplication)
```

Comment:
@lukehutch  Please help me check if the method `supportsLaunchMode` exists. I received this error when trying to build
Error (Xcode): ../../../../.pub-cache/git/flutter-maps-launcher-470dd6965ea83de929851c48723f5745dbafbfcd/lib/maps_launcher.dart:60:13: Error: Method not found: 'supportsLaunchMode'.

Reply:
Yes, this method is in `url_launcher`, which is a dependency of `flutter_maps_launcher`.

Reply:
I did update the deps versions though, as I said in the description, so make sure you update those, then dart `pub upgrade`